troposphere/static/css/app/app.scss

```diff
@@ -22,7 +21,7 @@
 //$navbar-default-link-color: darken(white, 10%);
 //$navbar-default-link-hover-color: white;
 //$navbar-default-link-active-color:         white;
-//$navbar-default-link-active-bg:            darken($navbar-default-bg, 6.5%);
+//$navbar-default-link-active-bg            darken($navbar-default-bg, 6.5%);
```

Comment:
Didn't mean to delete the `:` but I think I should just delete these commented out lines anyway.